.github/actions/aurora-manage-cluster/action.yml

```diff
@@ -0,0 +1,202 @@
+name: Deploy RDS Aurora Cluster
+
+description: |
+  This GitHub Action automates the deployment of an RDS Aurora cluster using Terraform.
+  This action will also install Terraform and awscli. It will output the Aurora cluster endpoint.
+
+inputs:
+  cluster-name:
+    description: 'Name of the RDS Aurora cluster to deploy'
+    required: true
+
+  engine-version:
+    description: 'Version of the Aurora engine to use'
+    required: true
+    # renovate: datasource=endoflife-date depName=amazon-rds-postgresql versioning=semver
+    default: "15.4"
+
+  instance-class:
+    description: 'Instance class for the Aurora cluster'
+    required: true
+    default: "db.t3.medium"
+
+  num-instances:
+    description: 'Number of instances in the Aurora cluster'
+    required: true
+    default: "1"
+
+  username:
+    description: 'Username for the PostgreSQL admin user'
+    required: true
+
+  password:
+    description: 'Password for the PostgreSQL admin user'
+    required: true
+
+  vpc-id:
+    description: 'VPC ID to create the cluster in'
+    required: true
+
+  subnet-ids:
+    description: 'List of subnet IDs to create the cluster in'
+    required: true
+
+  cidr-blocks:
+    description: 'CIDR blocks to allow access from and to'
+    required: true
+
+  auto-minor-version-upgrade:
+    description: 'If true, minor engine upgrades will be applied automatically to the DB instance during the maintenance window'
+    default: "true" # Default value from variable.tf
+
+  availability-zones:
+    description: 'Array of availability zones to use for the Aurora cluster'
+    default: ''
```

Comment:
the readme suggests the default is `[]` but here it's an empty string?
Does a user have to supply a comma separated string or an array?